[2023-01-19T10:00:37.625+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: pc_data_dag.clean_tables manual__2023-01-19T09:59:28.623487+00:00 [queued]>
[2023-01-19T10:00:37.635+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: pc_data_dag.clean_tables manual__2023-01-19T09:59:28.623487+00:00 [queued]>
[2023-01-19T10:00:37.636+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-01-19T10:00:37.637+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 6
[2023-01-19T10:00:37.637+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-01-19T10:00:37.650+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): clean_tables> on 2023-01-19 09:59:28.623487+00:00
[2023-01-19T10:00:37.657+0000] {standard_task_runner.py:55} INFO - Started process 2534 to run task
[2023-01-19T10:00:37.662+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'pc_data_dag', 'clean_tables', 'manual__2023-01-19T09:59:28.623487+00:00', '--job-id', '1831', '--raw', '--subdir', 'DAGS_FOLDER/create_dag_with_operator.py', '--cfg-path', '/tmp/tmpbc5trsq9']
[2023-01-19T10:00:37.664+0000] {standard_task_runner.py:83} INFO - Job 1831: Subtask clean_tables
[2023-01-19T10:00:37.743+0000] {task_command.py:389} INFO - Running <TaskInstance: pc_data_dag.clean_tables manual__2023-01-19T09:59:28.623487+00:00 [running]> on host 742349a45a57
[2023-01-19T10:00:37.812+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=malo
AIRFLOW_CTX_DAG_ID=pc_data_dag
AIRFLOW_CTX_TASK_ID=clean_tables
AIRFLOW_CTX_EXECUTION_DATE=2023-01-19T09:59:28.623487+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-19T09:59:28.623487+00:00
[2023-01-19T10:00:41.816+0000] {logging_mixin.py:137} INFO - connected
[2023-01-19T10:00:41.878+0000] {logging_mixin.py:137} INFO - table is truncated :companies
[2023-01-19T10:00:41.884+0000] {logging_mixin.py:137} INFO - table is truncated :foundedcompanies
[2023-01-19T10:00:41.886+0000] {logging_mixin.py:137} INFO - (psycopg2.errors.UndefinedTable) relation "industryempoyees" does not exist

[SQL: truncate table industryempoyees;]
(Background on this error at: https://sqlalche.me/e/14/f405)
[2023-01-19T10:00:41.887+0000] {logging_mixin.py:137} INFO - table not truncated
[2023-01-19T10:00:41.891+0000] {logging_mixin.py:137} INFO - table is truncated :countrycompanies
[2023-01-19T10:00:41.892+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-01-19T10:00:41.905+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=pc_data_dag, task_id=clean_tables, execution_date=20230119T095928, start_date=20230119T100037, end_date=20230119T100041
[2023-01-19T10:00:41.974+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-01-19T10:00:41.994+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
